zboxcore/sdk/sdk.go

```diff
@@ -76,6 +75,80 @@ func GetLogger() *logger.Logger {
 	return &Logger
 }
 
+type storageSdkSchema struct {
+	sdkInitialized bool
+}
+
+func NewStorageSdkSchema() *storageSdkSchema {
+	return &storageSdkSchema{}
+}
+
+var storageSdk = NewStorageSdkSchema()
```

Comment:
What's purpose of this variable?

Reply:
it is needed because in the first time, we have global `initializedSdk = true` and based on that, some file inside `zboxcore/sdk/` use it. So, instead of re-initializing the sdk every time we call a function, I created that global variable.